Changed artifacts version to 0.2.0-incubating

s8sg opened the pull request:
Update pom.xml: The Artifact version has been changed to 0.2.0-incubating from 0.2.0-incubating-SNAPSHOT
        Artifacts Source: https://mvnrepository.com/artifact/org.apache.beam

s8sg committed:
Changed artifacts version to 0.2.0-incubating

Update pom.xml: The Artifact version has been changed to 0.2.0-incubating from 0.2.0-incubating-SNAPSHOT
		Artifacts Source: https://mvnrepository.com/artifact/org.apache.beam

ecesena commented:
Hey, thanks for the contribution!

Sorry for the basic question, but did you try it on 0.2? I'm pretty confident there are breaking changes in FlinkRunner api so my code as is shouldn't work on 0.2.

Anyway, I'll have a look at it in the weekend and merge this. Thanks again!

s8sg committed:
Changes related to validation exception of Runner

pom.xml: Changed Start Class to WordCount as StreamWorkCount fails
StreamWordCount: removed validation
WordCount: removed validation

s8sg commented:
Hi @ecesena 
The WordCount example is working fine with the Above changes. i.e removing `.withValidation()`
But in StreamWordCount its getting the below Exception: 
```go
Caused by: java.lang.IllegalStateException: Unable to create a side-input view from input
	at org.apache.beam.sdk.transforms.View$AsIterable.validate(View.java:276)
	at org.apache.beam.sdk.transforms.View$AsIterable.validate(View.java:267)
	at org.apache.beam.sdk.Pipeline.applyInternal(Pipeline.java:400)
	at org.apache.beam.sdk.Pipeline.applyTransform(Pipeline.java:308)
	at org.apache.beam.sdk.values.PCollection.apply(PCollection.java:154)
	at org.apache.beam.sdk.io.Write$Bound.createWrite(Write.java:382)
	at org.apache.beam.sdk.io.Write$Bound.apply(Write.java:111)
	at org.apache.beam.sdk.io.Write$Bound.apply(Write.java:98)
	at org.apache.beam.sdk.runners.PipelineRunner.apply(PipelineRunner.java:76)
	at org.apache.beam.runners.flink.FlinkRunner.apply(FlinkRunner.java:140)
	at org.apache.beam.sdk.Pipeline.applyInternal(Pipeline.java:401)
	at org.apache.beam.sdk.Pipeline.applyTransform(Pipeline.java:324)
	at org.apache.beam.sdk.values.PCollection.apply(PCollection.java:167)
	at org.apache.beam.sdk.io.TextIO$Write$Bound.apply(TextIO.java:617)
	at org.apache.beam.sdk.io.TextIO$Write$Bound.apply(TextIO.java:463)
	at org.apache.beam.sdk.runners.PipelineRunner.apply(PipelineRunner.java:76)
	at org.apache.beam.runners.flink.FlinkRunner.apply(FlinkRunner.java:140)
	at org.apache.beam.sdk.Pipeline.applyInternal(Pipeline.java:401)
	at org.apache.beam.sdk.Pipeline.applyTransform(Pipeline.java:324)
	at org.apache.beam.sdk.values.PCollection.apply(PCollection.java:167)
	at com.dataradiant.beam.examples.StreamWordCount.main(StreamWordCount.java:144)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.flink.client.program.PackagedProgram.callMainMethod(PackagedProgram.java:505)
	... 35 more
Caused by: java.lang.IllegalStateException: GroupByKey cannot be applied to non-bounded PCollection in the GlobalWindow without a trigger. Use a Window.into or Window.triggering transform prior to GroupByKey.
	at org.apache.beam.sdk.transforms.GroupByKey.applicableTo(GroupByKey.java:173)
	at org.apache.beam.sdk.transforms.View$AsIterable.validate(View.java:274)
```
